src/index.js

```diff
@@ -184,10 +184,10 @@ class Plugin {
 		const alarmStatements = _.reduce(globalAlarms, (statements, alarm) => {
 			const key = this.naming.getAlarmCFRef(alarm.name, 'Global');
 			const cf = this.getAlarmCloudFormation(alertTopics, alarm, functionRefs);
-			statements[key] = cf;
 			if (alarm.pattern){
-				const logMetricCF = this.getLogMetricCF(alarm, functionRefs);
-				_.merge(statements, logMetricCF);
+				console.log('Note: global log metric filters are not supported');
```

Comment:
`serverless` log should be used. Check `compileCloudWatchAlarms` method, for example.